### ISSUE TITLE
Wav2lip only with DeepFaceLab

### ISSUE THREAD

**Issue by sktechnology** [iperov/DeepFaceLab#5498]
hello everyone,

Is it possible to do only wav2lip with the DeepFaceLab library, if yes, how to do it.

Thank you.

**Comment by ghost** [comment]
No, this will have to be done as a separate process. It can be combined in editing software for a final video. 
Issue solved / already answered (or it seems like user error), please close it.
